src/django/sqls/0151_index_number_of_workers.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE
+FUNCTION index_number_of_workers(af_id TEXT)
+RETURNS TABLE (number TEXT)
+LANGUAGE plpgsql
+AS $Body$
+BEGIN
+	RETURN QUERY
+SELECT
+    DISTINCT now_result
```

Comment:
Sorry if I am missing something, but we run distinct in each place where `index_number_of_workers` is called.

Reply:
<img width="1062" alt="Screenshot 2024-07-17 at 16 31 29" src="https://github.com/user-attachments/assets/6b726b0b-9985-4c0b-9d0c-9d67502c6f92">

I noticed that we call **distinct** after calling this function.

Reply:
oh I see your point, you are right probably.

Reply:
@vladsha-dev Fixed.